macros/base/end_print.cfg

```diff
@@ -9,32 +9,36 @@ gcode:
     {% set filter_enabled = printer["gcode_macro _USER_VARIABLES"].filter_enabled %}
     {% set light_enabled = printer["gcode_macro _USER_VARIABLES"].light_enabled %}
     {% set status_leds_enabled = printer["gcode_macro _USER_VARIABLES"].status_leds_enabled %}
+    {% set bed_mesh_enabled = printer["gcode_macro _USER_VARIABLES"].bed_mesh_enabled %}
 
     PARK
 
     {% if ercf_enabled %}
         # unload filament and park into ercf
-        _ERCF_FORM_TIP_STANDALONE
-        ERCF_UNLOAD
+        ERCF_EJECT
```

Comment:
This macro currently doesn't exist currently. Can you add it ?

Reply:
ERCF_EJECT is on the ercf.py script

Reply:
And depreciate in moggieuk scripts.

Reply:
Ohhh thanks. I don't own an ERCF at the moment and was not aware of that. Ok I'll merge :)